docs/sect-finality.md

```diff
@@ -485,3 +485,268 @@ A Catch-up response message contains critical information for the requester node
+
+:::
+###### Definition -def-num- Light Client {#defn-beefy-light-client}
+:::definition
+
+A **light client** is an abstract entity in a remote network such as Ethereum. It can be a node or a smart contract with the intent of requesting finality proofs from the Polkadot network. A light client reads the witness data ([Definition -def-num-ref-](sect-finality#defn-beefy-witness-data) from the chain, then requests the signatures directly from the relayer in order to verify those.
+
+:::
+###### Definition -def-num- Relayer {#defn-beefy-relayer}
+:::definition
+
+A **relayer** (or "prover") is an abstract entity that takes finality proofs from the Polkadot network and makes those available to the light clients. The relayer attempts to convince the light clients that the finality proofs have been voted for by the Polkadot relay chain validators. The relayer operates off-chain and can for example be a node or a collection of nodes.
+
+:::
+### -sec-num- Voting on Statements {#id-voting-on-statements}
+
+The Polkadot Host signs a statement ([Definition -def-num-ref-](sect-finality#defn-beefy-statement)) and gossips it as part of a vote ([Definition -def-num-ref-](chap-networking#defn-msg-beefy-gossip)) to its peers on every new finalized block. The Polkadot Host uses ECDSA for signing the statement since Ethereum has better compatibility for it compared to SR25519 or ED25519.
+
+### -sec-num- Committing Witnesses {#sect-beefy-committing-witnesses}
+
+The relayer ([Definition -def-num-ref-](sect-finality#defn-beefy-relayer)) participates in the Polkadot network by collecting the gossiped votes ([Definition -def-num-ref-](chap-networking#defn-msg-beefy-gossip)). Those votes are converted into the witness data structure ([Definition -def-num-ref-](sect-finality#defn-beefy-witness-data)). The relayer saves the data on the chain of the remote network. The occurrence of saving witnesses on remote networks is undefined.
+
+### -sec-num- Requesting Signed Commitments {#id-requesting-signed-commitments}
+
+A light client ([Definition -def-num-ref-](sect-finality#defn-beefy-light-client)) fetches the Signed Commitment Witness ([Definition -def-num-ref-](sect-finality#defn-beefy-witness-data)) from the chain. Once the light client knows which validators apparently voted for the specified statement, it needs to request the signatures from the relayer to verify whether the claims are actually true. This is achieved by requesting signed commitments ([Definition -def-num-ref-](chap-networking#defn-grandpa-beefy-signed-commitment)).
+
+How those signed commitments are requested by the light client and delivered by the relayer varies among networks or implementations. On Ethereum, for example, the light client can request the signed commitments in the form of a transaction, which results in a response in the form of a transaction.
```

Comment:
Should we keep this: "On Ethereum, for example, the light client can request the signed commitments in the form of a transaction, which results in a response in the form of a transaction."?

Reply:
Okay. Let's keep it as Motivation